src/MicrosoftTeams.schema.json

```diff
@@ -43,79 +43,136 @@
-                        "properties": {
-                            "44": {
+                            "people": {
                                 "type": "string",
-                                "description": "An icon for the extension sized to 44x44."
+                                "description": "A value of 'people' indicates the tab offers an experience in the context of a chat between one or more people"
                             },
-                            "88": {
+                            "personal": {
                                 "type": "string",
-                                "description": "An icon for the extension sized to 88x88."
+                                "description": "A value of 'personal' indicates the tab offers an experience scoped to an individual user alone"
                             }
-                        },
-                        "required": [
-                            "44",
-                            "88"
-                        ]
+                        }
+                    }
+                },
+                "required": [
+                    "configUrl",
+                    "scope"
+                ]
+            }
+        },
+        "staticTabs": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "entityId": {
+                        "type": "string",
+                        "description": "A unique identifier for the entity which the tab displays. The id must use reverse domain name notation.",
+                        "maxLength": 64
                     },
-                    "accentColor": {
+                    "name": {
                         "type": "string",
-                        "description": "A color to use in conjunction with the extension's icons. The value must be a valid HTML color code starting with '#', for example `#4464ee`."
+                        "description": "The display name of the tab.",
+                        "maxLength": 16
                     },
-                    "configUrl": {
+                    "canvasUrl": {
                         "type": "string",
-                        "description": "The url to use when configuring the extension."
+                        "description": "The url which points to the entity UI to be displayed in the Teams canvas."
                     },
-                    "canUpdateConfig": {
-                        "type": "boolean",
-                        "description": "A value indicating whether an instance of the extension's config can be updated by the user after creation.",
-                        "default": true
+                    "websiteUrl": {
+                        "type": "string",
+                        "description": "The url to point at if a user opts to view in a browser. If this is not supplied the browswer will be pointed at the canvasUrl"
```

Comment:
>If this is not supplied the browswer will be pointed at the canvasUrl [](start = 97, length = 69)

This isn't the case. We just don't show the "go to website" button if this isn't provided.

Note that this is the same as the websiteUrl in the tab settings when configuring a tab. See: [https://github.com/OfficeDev/microsoft-teams-library-js/blob/master/src/MicrosoftTeams.ts#L330](https://github.com/OfficeDev/microsoft-teams-library-js/blob/master/src/MicrosoftTeams.ts#L330)